napari/components/viewer_model.py

```diff
@@ -94,8 +94,28 @@ def __init__(
         self.mouse_move_callbacks = []
         # Hold callbacks for when mouse is pressed, dragged, and released
         self.mouse_drag_callbacks = []
+        # Hold callbacks for when mouse wheel is scrolled
+
+        def _scroll(viewer, event):
+            from .._viewer_key_bindings import (
+                increment_dims_left,
+                increment_dims_right,
+            )
+
+            if 'Control' not in event.modifiers:
+                return
+            delta = int(event.delta[1])
+            if abs(delta) >= 1:
+                for i in range(abs(delta)):
+                    if delta < 0:
+                        increment_dims_left(self)
+                    else:
+                        increment_dims_right(self)
```

Comment:
The last thing that needs to be done is that we need an accumulator for trackpad scrolling, which is fractional. So we would do something like:

```python
vertical_delta = event.delta[1]
int_delta = int(vertical_delta)  # vertical scrolling
fractional_delta = 
self._cumulative_delta += vertical_delta - int_delta
if self._cumulative_delta > 1:
    int_delta += 1
    self._cumulative_delta -= 1
```

... except there's two complicating factors:

- need to handle positive and negative deltas, which the above doesn't, and
- we need to reset cumulative delta to 0 after a certain time, or after any other event, since you don't really want the fractional scrolling from 3 minutes ago sitting around to modify your next scroll.

Probably I would say merge this without fractional deltas and then raise an issue for adding fractional deltas...